lib/elastic_apm/config/duration.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  class Config
+    # @api private
+    class Duration
+      MULTIPLIERS = { 'ms' => 0.001, 'm' => 60 }.freeze
```

Comment:
This indicates that the normalized value is `seconds`. The tests below also indicate seconds, but you actually name the value `minutes` here.